addons/event/i18n/event.pot

```diff
@@ -3365,6 +3365,22 @@ msgid ""
 "<b>and click Apply</b>."
 msgstr ""
 
+#. module: event
+#. openerp-web
+#: code:addons/event/static/src/js/tours/event_tour.js:0
+#, python-format
+msgid ""
+"Open date range picker. Pick a Start date for your event"
```

Comment:
Strange formatting, did you regenerate files through the UI ?

Reply:
I think(Hope) I generated the pot file correctly through the UI now :)